// try{
//     let result=10/1;
//     console.log(result);
// } catch(error){
//     console.log("An error occurred:",error.message);
// }

// let myFunction=(a,b)=>{
//     return(a*b);
// }
// console.log(myFunction(3,5));

hello= ()=>{
    return "Hello Workd!";
}
console.log(hello());